packages/manager/modules/overthebox/src/overthebox/actions/overTheBox-actions.html

```diff
@@ -0,0 +1,80 @@
+<div class="telecom-overTheBox-actions">
+    <div class="telecom-section-content">
+        <oui-message type="error" dismissable data-ng-if="$ctrl.isError">
+            <span data-ng-bind="$ctrl.errorMessage"></span>
+        </oui-message>
+
+        <div class="widget-presentation">
+            <h2
+                class="widget-presentation-title"
+                data-translate="overTheBox_actions_list"
+            ></h2>
+
+            <div
+                class="oui-datagrid-topbar__actions"
+                data-ng-if="!$ctrl.isLoading"
+            >
+                <button
+                    type="button"
+                    class="oui-button oui-button_secondary oui-button_icon-only oui-button_small-width"
+                    data-ng-click="$ctrl.getActions()"
+                    title="{{'common_refresh' | translate}}"
+                >
+                    <span class="fa fa-refresh" aria-hidden="true"></span>
```

Comment:
No `oui-icon` alternative ?

Reply:
After many tries, th oui-icon-refresh doesn't display correctly. Sorry

Reply:
What is the issue ? Maybe we can open an issue or try to fix the root cause to be able to use it instead ?

Reply:
For icon-refresh, the icon is blank

Reply:
Is `.oui-icon-refresh_extra-thin` helper can't be used?

Relates to: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/ovh-ui-kit/icons

Reply:
It's displayed with a very little size